File: client/src/app/tournament-results/model/event-results.ts
import {PlayerResults} from './player-results';

export class EventResults {

  // is single elimination round result
  singleElimination: boolean;

  // if true this is doubles event
  doubles: boolean;

  // round robin group number
  groupNumber: number;

  // round of 16, 8 , quarterfinals etc.
  round: number;

  playerResultsList: PlayerResults [];
}

File: server/src/main/java/com/auroratms/results/EventResults.java
package com.auroratms.results;

import lombok.Data;

import java.util.List;

@Data
public class EventResults {

    // is single elimination round result
    private boolean singleElimination;

    // if true this is doubles event
    private boolean doubles;

    // round robin group number
    private int groupNumber;

    // round of 16, 8 , quarterfinals etc.
    private int round;

    private List<PlayerResults> playerResultsList;
}
